content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -173,44 +230,83 @@ For more information on version skews, see:
    sudo apt-get install -y kubelet kubeadm kubectl
    sudo apt-mark hold kubelet kubeadm kubectl
    ```
+
 {{< note >}}
 In releases older than Debian 12 and Ubuntu 22.04, `/etc/apt/keyrings` does not exist by default.
 You can create this directory if you need to, making it world-readable but writeable only by admins.
 {{< /note >}}
 
 {{% /tab %}}
 {{% tab name="Red Hat-based distributions" %}}
-```bash
+
+### Prerequisites
```

Comment:
```suggestion
### {{% heading "prerequisites" %}} {#prerequisites-rpm}
```
(untested)


Another good option: go up to line 19 and add some details there, maybe with a link to a heading later in the file.

Reply:
I dropped this paragraph, I think it should be good enough without it, but let me know if you have any objections to this.